applications/SwimmingDEMApplication/custom_utilities/binbased_DEM_fluid_coupled_mapping.cpp

```diff
@@ -1439,7 +1406,7 @@ void BinBasedDEMFluidCoupledMapping<TDim, TBaseTypeOfSwimmingParticle>::Calculat
     double elemental_volume;
     GeometryUtils::CalculateGeometryData(geom, DN_DX, Ng, elemental_volume);
     const double& radius         = p_node->FastGetSolutionStepValue(RADIUS);
-    const double particle_volume = 4.0 * Globals::Pi / 3.0 * mParticlesPerDepthDistance * radius * radius * radius;
+    const double particle_volume = 4.0 * Globals::Pi / 3.0 * mParticlesPerDepthDistance * std::pow(radius, 3);
```

Comment:
Can ` mParticlesPerDepthDistance` be changed to another name? Like `particles_per_depth_distance` to make it coherent with Kratos style

Reply:
It is a member variable.